docs/reference/connectors.md

```diff
@@ -0,0 +1,91 @@
+# Connectors
+
+This resource packs supports the following connection configurations:
+
+* Environment Variables
+* Credentials File
+* Managed Service Identity Connector
+
+For any of these connection strategies you'll need a Subscription ID. Your Subscription ID may be found by ... When you run InSpec it will be included in the target. `-t azure://$SUBSCRIPTION_ID`.
+
+## Environment Variables
+
+Your account information will be read off of environment variables.
+
+On Linux:
+```
+export AZURE_CLIENT_ID=<Client ID>
+export AZURE_TENANT_ID=<Tenant ID>
+export AZURE_CLIENT_SECRET=<Client Secret>
+```
+
+On Windows:
+```
+$env:AZURE_CLIENT_ID="<Client ID>"
+$env:AZURE_CLIENT_SECRET="<Tenant ID>"
+$env:AZURE_TENANT_ID="<Client Secret>"
+```
+## Credentials File
+
+A credentials file may also be used. The default location is
```

Comment:
Its worth noting that if you set environment variables as stated above it will use those instead of the credentials file.

Reply:
That's a great idea. I'll add a section on precedence.